python/solid_dmft/main.py

```diff
@@ -96,9 +96,22 @@ def run_dmft(params, config_file_name=None):
     elif general_params['gw_embedding']:
         from solid_dmft.gw_embedding.gw_flow import embedding_driver
         if mpi.is_master_node():
+            # Checks for h5 file
+            if not os.path.exists(general_params['seedname']+'.h5'):
+                raise FileNotFoundError('Input h5 file not found')
```

Comment:
Why do we need to ensure that the h5 file is already there? I thought all quantities are obtained from the AIMBES chkpt file and we can start from an empty h5 file? Maybe I missed something, let me know.

Reply:
Since this PR replaces `dummy_sumk` by `SumkDFT` in the current GW embedding workflow (for automatic block_structure detection), `genera_params['seedname']+'.h5'` is necessary for the constructor of `SumkDFT`. Please correct me if I am wrong!

Reply:
ah yes of course! you are absolutely right. That makes sense. I just rerun all tests (including the GW ones, since the PR CI is not running them) and everything looks good. Can I merge it then?

Reply:
Yes, feel free to merge it. Thank you!